eblocker-ui/src/settings/app/components/openSourceLicenses/raw/javascript-dependencies.csv

```diff
@@ -151,7 +151,6 @@
 "from2","2.3.0","http://github.com/hughsk/from2","MIT"
 "fs-write-stream-atomic","1.0.10","https://github.com/npm/fs-write-stream-atomic","ISC"
 "fs.realpath","1.0.0","https://github.com/isaacs/fs.realpath","ISC"
-"fsevents","2.1.3","(none)","MIT"
```

Comment:
This change was made automatically by the maven build. Do these need to be checked in?